src/plone/api/content.py

```diff
@@ -16,18 +21,24 @@
 from zope.interface import Interface
 from zope.interface import providedBy
 
-import pkg_resources
 import random
 import transaction
 
 try:
-    pkg_resources.get_distribution('Products.Archetypes')
-except pkg_resources.DistributionNotFound:
+    get_distribution('Products.Archetypes')
+except DistributionNotFound:
     class IBaseObject(Interface):
```

Comment:
if you do this all objects will provide this interface, right? why not add a boolean variable stating whether you have or not Archetypes installed?